2022q2/pantheon.adoc

```diff
@@ -0,0 +1,24 @@
+=== Pantheon desktop on FreeBSD
+
+Links: +
+link:https://elementary.io/[elementary OS] URL: link:https://elementary.io/[https://elementary.io] +
+link:https://codeberg.org/olivierd/freebsd-ports-elementary[Development repository] URL: link:https://codeberg.org/olivierd/freebsd-ports-elementary[https://codeberg.org/olivierd/freebsd-ports-elementary] +
+
+Contact: Olivier Duchateau <duchateau.olivier@gmail.com>
+
+The Pantheon desktop environment is designed for elementary OS. It builds on GNOME technologies (such Mutter, GNOME Shell, GTK 3 and 4) and it is written with Vala language.
+
+The goal is to have new desktop for users. Some features are not well supported, but we can have full session.
+
+The repository contains Mk/Uses framework `elementary.mk`, official applications, and curated ports which depend of `x11-toolkits/granite` (total of 56 new ports).
+
+I have submitted several patches, especially:
+
+* `x11-toolkits/granite7`
+* `devel/libgee` update to 0.20.5 link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=262893[bug #262893]
+* `sysutils/bamf` update to 0.5.6 link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=264203[bug #264203]
+
+==== Open tasks
+
+* Add support of user settings (it is heavily targetted for Ubuntu)
```

Comment:
Maybe "very Ubuntu-centric" if you mean it expects to run in an Ubuntu environment and relies on it both for its UI and  for the settings it let you control?